lib/Config.ts

```diff
@@ -50,12 +51,13 @@ class Config {
   public logpath: string;
   public logdateformat: string;
   public network: XuNetwork;
-  public rpc: { disable: boolean, host: string, port: number };
-  public http: { host: string, port: number };
+  public rpc: { disable: boolean; host: string; port: number };
```

Comment:
As a matter of style, I'd prefer to keep things as is with regards to member delimiters here. It actually seems that [semicolons are preferred in typescript](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-delimiter-style.md) but I'd rather keep styling consistent and not change this as part of the PR.

Enforcing this as a linting rule looks like it'd require migrating from tslint to typescript-eslint, which I started working on but is not a trivial effort, so for now it'd be good to just leave these lines as is.